src/Jackett.Common/Utils/LoggingSetup.cs

```diff
@@ -17,14 +17,12 @@ public static LoggingConfiguration GetLoggingConfiguration(RuntimeSettings setti
         {
             var logFileName = settings.CustomLogFileName ?? "log.txt";
             var logLevel = settings.TracingEnabled ? NLog.LogLevel.Debug : NLog.LogLevel.Info;
-            // Add custom date time format renderer as the default is too long
-            ConfigurationItemFactory.Default.LayoutRenderers.RegisterDefinition("simpledatetime", typeof(SimpleDateTimeRenderer));
 
             var logConfig = new LoggingConfiguration();
 
             var logFile = new CleanseFileTarget
             {
-                Layout = "${longdate} ${level} ${message} ${onexception:inner=${newline}${newline}[v${assembly-version}] ${exception:format=ToString}${newline}${exception:format=Data}${newline}}",
+                Layout = "${longdate} ${level} ${onexception:[v${assembly-version}]${newline}}${message:withException=true}",
```

Comment:
Unfortunately these restore the duplicate first log line as before:

```text
08-10 00:36:05 Error Jackett.Common.IndexerException: Exception (cpasbien): The message received was unexpected or badly formatted.

[v0.21.602] Jackett.Common.IndexerException: Exception (cpasbien): The message received was unexpected or badly formatted.
```

Is there a benefit, or something this would resolve that I'm missing?